Added support for Brotli compression

bstordrup opened the pull request:
Adding support for Brotli compressio.

Fixes #54

bstordrup committed:
Added support for Brotli compression

bstordrup commented:
Unfortunately, I don't have a pcap file containing Brotli encoded material, so I'm not able to provide unit test for this.

chmorgan commented:
Hi @bstordrup, how do we know this is the right approach for decoding this compression? I think we would need a test stream, even if it was a pcap stream from somewhere else that we could use here. Thoughts on where we might get one?

bstordrup commented:
@chmorgan, I know of a page that supports compressing the result with Brotli. I can see if I can capture something from it with Wireshark.

bstordrup committed:
Added unit test for brotli handling

bstordrup commented:
Added a pcap file that contains a HTTP GET request to an REST API project and returns a brotli compressed response in form of a Json file.

And added a unit test that evaluates this pcap file in the same way as handling the gziped contents pcap.

chmorgan commented:
@bstordrup sorry for leaving this so long. If this is still good I'd like to merge it in. Can you confirm its still ok?
